framework/generated/khronos_generators/khronos_base_generator.py

```diff
@@ -463,6 +463,37 @@ def beginFeature(self, interface, emit):
         if self.genOpts.protect_feature:
             self.featureExtraProtect = self.get_feature_protect(interface)
 
+        if not self.generate_video:
+            self.gen_video_type()
+            self.generate_video = True
+
+    def gen_video_type(self):
+        if not self.VIDEO_TREE:
+            return
+
+        if self.process_structs:
+            types = self.VIDEO_TREE.find('types')
+            for element in types.iter('type'):
+                name = element.get('name')
+                category = element.get('category')
+                if name and category and (category == 'struct' or category == 'union'):
+                    self.struct_names.add(name)
+                    if category == 'struct':
+                        self.feature_struct_members[name] = self.make_value_info(
+                            element.findall('member')
+                        )
+
+        for element in self.VIDEO_TREE.iter('enums'):
+            group_name = element.get('name')
+            self.enum_names.add(group_name)
+            enumerants = dict()
+            for elem in element.findall('enum'):
+                name = elem.get('name')
+                value = elem.get('value')
+                if value:
+                    enumerants[name] = value
+            self.enumEnumerants[group_name] = enumerants
+
```

Comment:
I'm curious why this entire block is part of this PR, it doesn't seem to affect the vulkan output...

Reply:
@jzulauf-lunarg is this what you meant by stray code? https://github.com/LunarG/gfxreconstruct/pull/1920#pullrequestreview-2512985931

Reply:
I'm not 100% sure how this code got here. I imagine I messed something up during the rebase I had to perform as a part of this PR. From what I can tell from the git history, this code _should_ just be removed.